adapters/sovrn/sovrntest/supplemental/gdpr.json

```diff
@@ -20,7 +20,8 @@
           "bidder": {
             "tagid": "123456"
           }
-        }
+        },
+        "tagid": "123456" 
```

Comment:
Why is this field added to so many things?

Reply:
Because we originally added it to the Sovrn EXT object in prebid, but it needs to be copied to its correct location in the `imp` object.

Reply:
"Partial OpenRTB" indeed.

Reply:
Does this mean that we are expecting/requiring this field to be populated somewhere? Or we are now ensuring that our PBS adapter will populate this field?

Reply:
The adapter-specific ext objects are managed by prebid server itself. That was a bit of confusion I had to figure out when first writing this. Since the required `video` values are in the openrtb request, I had to write code to do the checking of the required values.